sdk/servicebus/service-bus/src/models.ts

```diff
@@ -103,6 +104,16 @@ export interface MessageHandlerOptions {
  */
 export interface CreateSessionReceiverOptions extends SessionReceiverOptions, OperationOptions {}
 
+/**
+ * Describes the options passed to the `createSender` method on `ServiceBusClient`.
+ */
+export interface CreateSenderOptions {
+  /**
+   * The signal which can be used to abort requests.
+   */
+  abortSignal?: AbortSignalLike;
```

Comment:
Just a question that doesn't need to be resolved now, but are we planning to have retry options here as well?

Reply:
That should be discussed with the broader team.